docs/release_docs/AHA_Platform_Deployment_Workflows.md

```diff
@@ -0,0 +1,71 @@
+mode of trigger: ```On-Demand```
+
+# AHA Platform Deployment Workflows
+
+## AHA-PROD-CI-CD
+
+This workflow uses two jobs: GitHub-ECR-Tag-Check and Deploy ECS to verify and deploy ```reancare-service``` release to ```aha-prod``` environment
+
+### JOBS
+
+#### GitHub-ECR-Tag-Check
+The GitHub ECR Tag Check will be performing the following steps.
+
+* This job uses [mukunku/tag-exists-action](https://github.com/marketplace/actions/tag-exists-action), [git-get-release-action](https://github.com/marketplace/actions/git-get-release-action).
+* This job will validate whether the given input release tag exists or not
+* This job get the GitHub release with the associated GitHub tag name with it and store the GitHub release ID.
+* Then it will check ECR image tag with the same as GitHub release ID.
+
+#### Deploy-ECS
+The Deploy ECS will be performing the following steps
+
+* This job uses [docker/build-push-action](https://github.com/marketplace/actions/build-and-push-docker-images).
+* This job uses 'aha-prod' environment and login to ECR using creds and pull the ECR image which was created in PROD-CI-CD with the GitHub release ID.
+* Then it will create new version of Amazon ECS task definition with new docker image and deploy Amazon ECS task definition using Duplo API.
+
+## AHA Prod Release Workflow
```

Comment:
This heading should not be second level heading as it conflicts with the above second level heading "AHA-PROD-CI-CD" and makes it looks like its another section while it is a part of the same section